ddl/ddl.go

```diff
@@ -551,6 +560,38 @@ func (d *ddl) GetHook() Callback {
 	return d.mu.hook
 }
 
+func (d *ddl) startCleanDeadTableLock() {
+	defer goutil.Recover(metrics.LabelDDL, "startCleanDeadTableLock", nil, false)
+
+	ticker := time.NewTicker(time.Second * 10)
+	defer ticker.Stop()
+	for {
+		select {
+		case <-ticker.C:
+			if !d.ownerManager.IsOwner() {
+				continue
+			}
+			deadLockTables, err := d.tableLockCkr.GetDeadLockedTables(d.infoHandle.Get().AllSchemas())
```

Comment:
Could we add the argument of `context` to `GetDeadLockedTables ` to control when the function to exit?

Reply:
great, done.